oci/main.py

```diff
@@ -43,15 +43,66 @@ def run():
              'under "ENGINE_VERSION"',
         default='master')
 
+    stage_parser = subparsers.add_parser(
```

Comment:
For consistency with other code, lets call this run_parser.

Reply:
Makes sense. Done.